c/zos.c

```diff
@@ -1029,12 +1058,14 @@ static int safVerifyInternal(int options,
     verifyRequest->initflg3 = verifyFlags3;
 
     if (safTrace){
-      fprintf(safTraceFile,"countedUserid %0.8X' password %0.8X' passphrase %0.8X'\n",
+      fprintf(safTraceFile,"countedUserid 0x%p' password 0x%p' passphrase 0x%p'\n",
               countedUserid, countedPassword, countedPassphrase);
-      fprintf(safTraceFile,"about to go call saf wrapper at %0.8X' verify block at %0.8X' acee %0.8X'\n",safWrapper,verifyRequest,
+      fprintf(safTraceFile,"about to go call saf wrapper at 0x%p' verify block at 0x%p' acee 0x%p'\n",safWrapper,verifyRequest,
               (aceeHandle != NULL ? *aceeHandle : NULL));
       dumpbuffer((char*)safWrapper,sizeof(safp));
-      fprintf(safTraceFile,"verify:\n",0);
+#ifndef METTLE
```

Comment:
Why do we need to shield this calls against Metal and not the `fprintf` calls above?